### DISCUSSION TITLE
New: Host name or IP address

### DISCUSSION THREAD

**Discussion by o0OKAASKOPO0o** [plmilord/Hass.io-custom-component-ikamand#16] [General]
Thanks for time and effort you put into this.
I got Home assistant, got HACS. But when I want to open the HACS Ikamand I get a screen to put the host name or IP-address. I have no clue what IP address I have to fill in. Thanks for the help.

**Comment by o0OKAASKOPO0o** [comment]
![Screenshot_20241029_131354_Home Assistant](https://github.com/user-attachments/assets/56a3aa6b-1ab1-4cb9-97cd-3de9ac5797d5)

**Comment by plmilord** [comment]
You should be able to find your iKamand IP address in your wifi router. Just log in as admin and see who's logged in!

**Reply by o0OKAASKOPO0o** [reply]
Thanks for the fast response. I thought this was the software to connect IKamand with my local wifi netwerk.
But with what software/app do you connect the iKamand with your phone/laptop and do you send de SSID and password. What software did you use to make [this screenshot.](https://raw.githubusercontent.com/plmilord/Hass.io-custom-component-ikamand/master/images/connect.png)

**Reply by plmilord** [reply]
I used my iPhone with the iKamand app to take this screenshot. You don't need this app to use this custom component, just follow this step-by-step procedure!

**Reply by o0OKAASKOPO0o** [reply]
But the ikamand app server is offline, the app isnt supported anymore. I thought this was the workaround to still use the iKamand hardware with other software.

**Comment by plmilord** [comment]
As stated, no need for this iKamand app... Just follow what is written in the instructions ([this screenshot](https://raw.githubusercontent.com/plmilord/Hass.io-custom-component-ikamand/master/images/connect.png))

**Reply by krizzziz** [reply]
> As stated, no need for this iKamand app... Just follow what is written in the instructions ([this screenshot](https://raw.githubusercontent.com/plmilord/Hass.io-custom-component-ikamand/master/images/connect.png))

I have read you comment about a 100x times and was not able to figure out what I was doing wrong. 
Indeed you must to exactly what the images states, but what is logical to do for others. Was not so logical for me. After 100x attempts it worked :)

@o0OKAASKOPO0o stuur me een berichtje als het niet lukt (send me a message when it's not working). I succeeded this night and brought my Ikamand back to life. WOW.

**Reply by o0OKAASKOPO0o** [reply]
@krizzziz thanks for your comment. It still doesn't work for me. I can connect to  Ikamand with the wifi. But the 192.168.10.1/cg0-bin/netsetdoesn't load. so i cant fill in mu ssid to connect to my local netwerkt to setup for home assist

**Reply by krizzziz** [reply]
> @krizzziz thanks for your comment. It still doesn't work for me. I can connect to Ikamand with the wifi. But the 192.168.10.1/cg0-bin/netsetdoesn't load. so i cant fill in mu ssid to connect to my local netwerkt to setup for home assist

This worked for me first time I tried it, so that's indeed different. you did try 192.168.10.1/cgi-bin/netset right? The adress you wrote does absolutely nothing. You fill in the adres on your browser on your phone right?
